velox/expression/tests/SignatureBinderTest.cpp

```diff
@@ -30,6 +30,26 @@ void testSignatureBinder(
   ASSERT_TRUE(expectedReturnType->kindEquals(returnType));
 }
 
+TEST(SignatureBinderTest, decimals) {
+  // decimal(10, 3), decimal(10, 5) -> decimal()
+
+  auto signature =
+      exec::FunctionSignatureBuilder()
+          .returnType("SHORT_DECIMAL(r_precision, r_scale)")
+          .argumentType("SHORT_DECIMAL(a_precision, a_scale)")
+          .argumentType("SHORT_DECIMAL(b_precision, b_scale)")
+          .variableConstraint(
+              "r_precision",
+              "min(b_precision - b_scale, a_precision - a_scale) + max(a_scale, b_scale)")
```

Comment:
Would it make sense to add tests for the actual expressions that will be used for decimal operators to make sure we have enough support?

- add

```
 longVariableExpression("r_precision", "min(38, max(a_precision - a_scale, b_precision - b_scale) + max(a_scale, b_scale) + 1)"),
 longVariableExpression("r_scale", "max(a_scale, b_scale)"))
```

- subtract

```
 longVariableExpression("r_precision", "min(38, max(a_precision - a_scale, b_precision - b_scale) + max(a_scale, b_scale) + 1)"),
 longVariableExpression("r_scale", "max(a_scale, b_scale)"))
```

- multiply

```
 longVariableExpression("r_precision", "min(38, a_precision + b_precision)"),
 longVariableExpression("r_scale", "a_scale + b_scale"))
```

- divide, modulus

Reply:
Yeah, will add all these.